src/index.css

```diff
@@ -0,0 +1,18 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
```

Comment:
Instead of placing your components' styling in this file kindly just make a separate CSS file for each:

`App.css`
`ButtonPanel.css`
`Button.css`
`Display.css`